dtorrent/src/main.rs

```diff
@@ -0,0 +1,34 @@
+use std::{env};
+use std::path::{PathBuf};
+use std::{sync::Arc};
+use std::collections::HashMap;
+use dtorrent::{bt_server::server::BtServer, config::cfg::Cfg, torrent_parser::{parser::TorrentParser}, torrent_handler::status::AtomicTorrentStatus};
+use logger::logger_receiver::Logger;
+
+fn main() {  
+    // Reads the filepath from the command line argument (Check README)
+    let mut arg = env::args();
+    let _ = arg.next().expect("Failed to read command line arguments");
+    let path = arg.next().expect("Failed to retrieve file path");
+    let path = PathBuf::from(path.trim());
+  
```

Comment:
Check out [skip(n)](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.skip) and [nth(n)](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.nth) which may make this prettier, for example, if you know the path will always be the 1st argument (counting from 0), then `nth(1)` is perfect here.